contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
added ContribX people.  we don't have an Aliases file yet.  I'll work on that in a separate PR.